src/renderer/components/Wizard/Details/WizardDetails.tsx

```diff
@@ -13,6 +14,7 @@ export const WizardDetails = () => {
     const [author, setAuthor] = useState('');
```

Comment:
Remember to add the project properties as initial values if they exist.

Reply:
Currently do on mount of the component if the project already has values 
![image](https://user-images.githubusercontent.com/6975420/114321421-b97d0300-9ae8-11eb-987f-0c3664de2a4d.png)

Reply:
👍